compiler-rt/lib/sanitizer_common/sanitizer_stacktrace.cpp

```diff
@@ -110,14 +110,15 @@ void BufferedStackTrace::UnwindFast(uptr pc, uptr bp, uptr stack_top,
   while (IsValidFrame((uptr)frame, stack_top, bottom) &&
          IsAligned((uptr)frame, sizeof(*frame)) &&
          size < max_depth) {
-#ifdef __powerpc__
-    // PowerPC ABIs specify that the return address is saved at offset
-    // 16 of the *caller's* stack frame.  Thus we must dereference the
-    // back chain to find the caller frame before extracting it.
+#  ifdef __powerpc__
```

Comment:
trivial code/comment moves  like this either unnecessary or should go into separate [NFC] PRs

Reply:
can we then replace regular name with display name?

Reply:
> can we then replace regular name with display name?

Can you elaborate on what you mean? If we say that the "primary name" is what was already in the code before this patch, are you saying that you prefer for the implementation on AIX to use the display name as the "primary name" (and for AIX to have an additional "object path" that it uses when invoking the symbolizer)?